<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>12</y>
      <width>1221</width>
      <height>721</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QStackedWidget" name="MainQSW">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="NProcesosQSW">
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>480</y>
           <width>481</width>
           <height>161</height>
          </rect>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Elige el número de procesos a ingresar</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="NProcesosQSB">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="accelerated">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="Aceptar">
            <property name="text">
             <string>Aceptar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>30</y>
           <width>401</width>
           <height>401</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="applogo.qrc">:/images/logo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="EjecProcesosQSW">
        <widget class="QTableWidget" name="PendientesQTV">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>270</y>
           <width>291</width>
           <height>431</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>180</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>TME</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>TT</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
        <widget class="QLabel" name="NoProcPenQL">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>331</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Número de procesos nuevos:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="NoProcPendientesQLE">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>10</y>
           <width>81</width>
           <height>24</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="LoteEjecQL">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>240</y>
           <width>181</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Procesos Listos:</string>
         </property>
        </widget>
        <widget class="QLabel" name="ProcTerml">
         <property name="geometry">
          <rect>
           <x>840</x>
           <y>240</y>
           <width>211</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Procesos terminados:</string>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>1121</width>
           <height>161</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="LoteEjecQL_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Proceso Actual:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="IDActualQL">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="IDActualQLE">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>24</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="OperacionActualQL">
              <property name="maximumSize">
               <size>
                <width>110</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Operación:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="OperacionActualQLE">
              <property name="maximumSize">
               <size>
                <width>800</width>
                <height>24</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="TMEQL">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>TME:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="TMEActualQLE">
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>24</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="TiempoRestanteQL">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>TR:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="TiempoRestanteQLE">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>24</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="TiempoTranscurridoQL">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>TT:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="TiempoTranscurridoQLE">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="ContadorQL">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>10</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Contador:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="ContadorQLE">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>10</y>
           <width>81</width>
           <height>24</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QTableWidget" name="ProcTerminadosQTW">
         <property name="geometry">
          <rect>
           <x>710</x>
           <y>270</y>
           <width>421</width>
           <height>431</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Operacion</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Resultado</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
        <widget class="QLabel" name="ProcBloqueadosQL">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>240</y>
           <width>221</width>
           <height>31</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>Procesos Bloqueados:</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="BloqueadosQTW">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>270</y>
           <width>331</width>
           <height>431</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tiempo Bloqueado</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
        <widget class="QPushButton" name="VerTiemposQPB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>980</x>
           <y>0</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Ver Tiempos</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QTableWidget" name="TiemposQTW">
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>ID</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Operacion</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Resultado</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>T Llegada</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>T Finalizacion</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>T Retorno</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>T Respuesta</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>T Espera</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>T Servicio</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0">
            <widget class="QTableWidget" name="BCPQTW">
             <property name="tabKeyNavigation">
              <bool>true</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Estado</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Operacion</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Resultado</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T Llegada</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T Fin</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T Retorno</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T Espera</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T Servicio</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T Restante</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T Respuesta</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>TR Bloqueado</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
            </widget>
           </item>
           <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>BCP PROCESOS</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1263</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="applogo.qrc"/>
 </resources>
 <connections/>
</ui>
